assets/scripts/cart.js

```diff
@@ -0,0 +1,200 @@
+import { create, main } from "./aggregate.js";
+
+function cart() {
+  const main = document.querySelector(".wrapper");
+  const fragment = document.createDocumentFragment();
+  const cartField = create("div", "cartField", fragment, "");
+  cartField.id = "cartField";
+  cartField.classList.add("droppable");
+  const list = create("ul", "list", cartField);
+  const total = create("div", "liRow", cartField);
+  total.classList.add("total");
+  if (document.title === "Bookshop") {
```

Comment:
I would say it's better to move the code into a separate function.
And call each instance inside the initializing function. It helps split your code into smaller pieces that are easier for understanding.
For instance: 
```suggestion
 const initCart = () => {
   renderCart();
   if(smth) {
      renderCartConfirmationBtn();
   }
   
   ....
   renderCartDiscountSection();
   renderCartAdvertisiment() 
   ... and so on
 }
```